# all cones treated the same
uint8 CUBE=0
uint8 VERTICAL_CONE=1
uint8 BASE_TOWARDS_US_CONE=2 
uint8 BASE_AWAY_US_CONE=3
uint8 CLOSEST_PIECE=4
uint8 piece
---
float64 nearest_cone_angle
float64 nearest_cube_angle
geometry_msgs/Point cube_point
geometry_msgs/Point cone_point
bool success